### ISSUE TITLE
Is 0.2.7 backwards compatible with 0.2.6?

### ISSUE THREAD

**Issue by aclysma** [SSheldon/rust-objc#87]
Hello, I had trouble compiling cocoa when using 0.2.7, but was able to use 0.2.6. 

```
error[E0282]: type annotations needed
   --> /Users/username/.cargo/registry/src/github.com-1ecc6299db9ec823/cocoa-0.13.0/src/appkit.rs:491:9
    |
491 |         msg_send![self, releaseGlobally];
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |         |
    |         cannot infer type
    |         consider giving `result` a type
    |
    = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)
```

My current workaround is to use `objc = ">=0.2,<0.2.7"` in my cargo.toml. I'm honestly not sure why it fails to build, but I didn't expect that 0.2.6 -> 0.2.7 would be a breaking change. I have an issue open in my project https://github.com/aclysma/skulpin/issues/6 - just wanted to check that this is the best solution?

Thanks for all the work on this project!

**Comment by SSheldon** [comment]
@aclysma sorry for the inconvenience! 0.2.7 has a soundness fix to prevent future undefined behavior. If it's failing to compile, that means your dependencies could exercise undefined behavior in a future version of rust. There's more info on context and reasoning in #62.

I'll see if I can give you any advice on how to fix this over at aclysma/skulpin#6.

**Comment by aclysma** [comment]
Thanks so much for the help!
